src/main/java/com/google/api/codegen/config/GapicProductConfig.java

```diff
@@ -429,8 +471,15 @@ public static GapicProductConfig createDummyInstance(
   private static ImmutableMap<String, SingleResourceNameConfig> createSingleResourceNameConfigs(
       DiagCollector diagCollector,
       ConfigProto configProto,
-      ProtoFile file,
-      TargetLanguage language) {
+      @Nullable List<ProtoFile> sourceProtos,
+      TargetLanguage language,
+      ProtoParser protoParser) {
+    ProtoFile file = null;
```

Comment:
Is this variable used?

Reply:
It's used several call stacks deeper, in `FixedResourceNameConfig.createFixedResourceNameConfig()`, `SingleResourceNameConfig.createSingleResourceName()`, `ResourceNameOneofConfig.createResourceNameOneof()`, and possibly something else.

Reply:
Oh I somehow missed those lines.